src/test/java/jewellery/inventory/integration/ResourceInOrganizationCrudIntegrationTest.java

```diff
@@ -49,6 +51,76 @@ private String getBaseResourceUrl() {
   }
 
   @Test
+  void transferResourceToOrganizationSuccessfully() throws JsonProcessingException {
```

Comment:
again, please, keep the test in 3 sections (preparation, action, assertions) with a single empty line between the sections.